plugins/generator-1.17.1/forge-1.17.1/triggers/entity_joins_world.java.ftl

```diff
@@ -0,0 +1,12 @@
+@Mod.EventBusSubscriber public class ${name}Procedure {
+	@SubscribeEvent public static void onEntityJoin(EntityJoinWorldEvent event) {
+		Entity entity=event.getEntity();
+		Map<String, Object> dependencies = new HashMap<>();
+		dependencies.put("x", entity.getX());
+		dependencies.put("y", entity.getY());
+		dependencies.put("z", entity.getZ());
+		dependencies.put("world",entity.level);

```

Comment:
At this point. level of entity may not be populated yet, thus event provides event.getWorld()

Reply:
I copied a template with x,y,z and world dependencies, but I didn't check carefully the code giving the value. Sorry 😬